### PR TITLE
feat: add tests cases for  chaoscenter web views

### PR DESCRIPTION
**Pull request by Freedisch**
## Proposed changes

Added test cases for views in chaoscenter web 
- ErrorCheck.tsx
- ResestPassowrd.tsx
- Overview.tsx

image

## Types of changes

What types of changes does your code introduce to Litmus? Put an `x` in the boxes that apply
- [x] New feature (non-breaking change which adds functionality)
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update (if none of the other choices applies)

## Checklist

Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.
- [ ] I have read the [CONTRIBUTING](https://github.com/litmuschaos/litmus/blob/master/CONTRIBUTING.md) doc
- [ ] I have signed the commit for DCO to be passed.
- [ ] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works (if appropriate)
- [ ] I have added necessary documentation (if appropriate)

## Dependency
- Please add the links to the dependent PR need to be merged before this (if any).

### CODE REVIEW

**Comment by Freedisch** [comment]
image

File: chaoscenter/web/src/views/ResetPassword/__tests__/ResetPassword.test.tsx [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+
+  beforeEach(() => {
+    jest.spyOn(stringUtils, 'useStrings').mockReturnValue({
+      getString: jest.fn().mockImplementation(key => `Mocked String for ${key}`)
+    });
+  });
+
+  afterEach(() => {
+    jest.clearAllMocks();
+  });
```
Comment (S-ayanide) [inline chaoscenter/web/src/views/ResetPassword/__tests__/ResetPassword.test.tsx:20]: Same review as before

**Review by S-ayanide — CHANGES_REQUESTED** [review]

File: chaoscenter/web/src/utils/testUtils.tsx [hunk 2]
```diff
@@ -2,39 +2,55 @@
...
-          updateAppStore: () => void 0
-        }}
-      >
-        <StringsContext.Provider value={{ data: strings as any, getString }}>{children}</StringsContext.Provider>
-      </AppStoreContext.Provider>
-    </BrowserRouter>
+    <ReactQueryProvider>
+      <ApolloProvider client={appoloClient}>
+        <BrowserRouter>
+          <AppStoreContext.Provider
```
Comment (S-ayanide) [inline chaoscenter/web/src/utils/testUtils.tsx:36]: AppStoreContext Provider should be the top most in the order, here is the revised order for reference

```tsx
<AppStoreContext.Provider>
  <StringContext.Provider>
    <ReactQueryProvider>
       <ApolloProvider>
          <BrowserRouter />
       </ApolloProvider>
    </ReactQueryProvider>
  </StringContext.Provider>
</AppStoreContext.Provider>
```

File: chaoscenter/web/src/views/ResetPassword/__tests__/ResetPassword.test.tsx [hunk 3]
```diff
@@ -0,0 +1,85 @@
+/* eslint-disable @typescript-eslint/no-empty-function */
```
Comment (S-ayanide) [inline chaoscenter/web/src/views/ResetPassword/__tests__/ResetPassword.test.tsx:1]: Please remove this eslint disable statement and fix any linting issues.

**Review by S-ayanide — CHANGES_REQUESTED** [review]

**Review by hrishavjha — APPROVED** [review]
Thanks for the change, apart from @S-ayanide 's comments rest looks good to me. Please address them🚀

**Review by S-ayanide — APPROVED** [review]
LGTM

**Pull request merged**